lib/plugin.dart

```diff
@@ -358,6 +358,13 @@ class AtomDartPackage extends AtomPackage {
         'enum': ['all', 'uncaught', 'none'],
         'order': 8
       },
+      'debuggerDdcParsing': {
+        'title': "Map web debug stack to DDC output",
```

Comment:
Hmm, 'Convert JavaScript stack traces to Dart'? 'Display compiled JavaScript stack traces as original Dart locations'?

Reply:
Done.